src/internal/data-service-api/data-service-api.ts

```diff
@@ -192,6 +202,13 @@ export interface DataServiceDappsApi {
   findAllDappAddresses(): Promise<DappAddressDto[]>;
 }
 
+function createHeaders(token: Token) {
+  return {
+    Authorization: `Bearer ${token.rawValue}`,
+    [XRequestIdHeader]: crypto.randomUUID(),
```

Comment:
crypto is a node specific api. can we use something else, to support browser environments and mobile without using polyfills/shims?

Reply:
you can take a look at https://github.com/uuidjs/uuid
or https://github.com/ai/nanoid if you don't really need uuids

Reply:
agreed, missed this, fixed